services/api/src/messaging/players.service.ts

```diff
@@ -18,6 +18,16 @@ const figureMoved: Handler = (ws, payload): void => {
   publish({ type: MessageType.NewBoardState, payload: newBoardState });
 };
 
+const moveSkipped: Handler = (ws, payload): void => {
+  const { userId } = payload as { userId: string };
+  const [playerConnection, player] = Array.from(players.entries()).find(([_, player]) => player.id === userId && player.status === PlayerStatus.ActionNotTaken) || [null, null];
```

Comment:
My guess is that finding player by id will be quite common operation across the API, so I would suggest to introduce a separate method for this early on to improve readability.

Also, maybe it would make sense to separate player lookup by id and player status validation to be in separate actions so that we could handle those separately as they most likely should be handled independently:
```
   const [playerConnection, player] = findPlayerById(players, userId);
   if (!player) {
      throw new Error(`Player with id ${userId} not found`)
   }
   
   if (player.status !== PlayerStatus.ActionNotTaken) {
      throw new Error(`Player ${userId} cannot skip a move`)
   }
   ...
```